src/add-remove.test.js

```diff
@@ -0,0 +1,20 @@
+const myModules = require('./add-remove.js');
```

Comment:
- [ ] The previous reviewer may forget to mention this but your PR has no description, Kindly add a descriptive one so your PR can be validated